src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Context } from '../../constants';
+
+export interface Props extends React.HTMLAttributes<HTMLDivElement> {
+    /** Short and descriptive title */
+    title: string;
+    /** Body content */
+    content?: string;
+    /** The Toast context (e.g. info, critical, success or cautious - defaults to info) */
+    context?: Context;
+    /** If there are actions (links or buttons) in the toast */
+    actionable?: boolean;
+    /** Action object, containing item text and href to action */
+    actionOne?: { text: string; href: string };
+    /** Action object, containing item text and href to action */
+    actionTwo?: { text: string; href: string };
```

Comment:
Maybe instead of going with 1 and 2, we could have it as one prop `actions?: {text: string; callback: () => void; href: string}[]`. Also note that I added `callback` here. I think an action can be a function not only a navigation item. We can use some `if` statement maybe to see if we need to render a link (`<a>`) or a button component

Reply:
Yes, very good idea. Should we put a limit on the amount of actions, though?

Reply:
Please check with Dima, but my 2 cents is there that would be over engineering. Let's hope people are resonable enough not to offer 30 actions to a user in the context of a toast.
Also I don't know how would you limit that in the TS definition. In the code you could make sure to render only the first few, but that is really opinionated, and sounds like a source of bugs to me :-D

Reply:
There should indeed be a limit of 2 actions according to Dima. We could still keep the actionOne & actionTwo, just adding the callback. But a single prop does look cleaner indeed

Reply:
I'd go with the array, and maybe say in the prop description that more then 2 actions is not recommended. But I wouldn't enforce it